readthedocs/config/config.py

```diff
@@ -296,6 +295,28 @@ def get_valid_python_versions_for_image(self, build_image):
             )
         return settings.DOCKER_IMAGE_SETTINGS[build_image]['python']['supported_versions']
 
+    def get_default_python_version_for_image(self, build_image, python):
+        """
+        Return the default Python version for Docker image and Py2 or Py3.
+
+        :param build_image: the Docker image complete name, already validated
+            (``readthedocs/build:4.0``, not just ``4.0``)
+        :type build_image: str
+
+        :param python: major Python version (``2`` or ``3``) to get its default
```

Comment:
This should probably be `python_version`, otherwise below its confusing when used as `[build_image]['python']['default_version'][python]`